css/main.css

```diff
@@ -207,3 +202,74 @@ hr {
   border-bottom: 1px solid var(--neutral);
   margin: 12px auto;
 }
+
+/* contact form start here */
+.contact-form-container {
+  width: 100%;
+  background: url('../images/contact-form.png'), var(--logo-purple);
+  background-repeat: no-repeat;
+
+  /* background-attachment: fixed; */
+  background-size: 100% 100%;
+  border-top-left-radius: 50px;
+  height: auto;
+  padding: 60px 0;
+}
+
+.contact-form-container .contact-form {
+  max-width: 447px;
+  text-align: center;
+  margin: auto;
+  padding: 50px 18px;
+}
+
+.contact-form h2 {
+  color: #fff;
+  margin: 30px 0 12px;
+}
+
+.contact-form p {
+  font-weight: 400;
+  font-size: 1.25rem;
+  color: #ebeffe;
+  margin: 10px 0 80px;
+}
+
+.contact-form .form {
+  flex-direction: column;
+  justify-content: center;
+  gap: 20px;
+}
+
+.contact-form .input {
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  border-radius: 8px;
+  font-weight: 400;
+  font-size: 1.06rem;
+  color: #172b4d;
+  width: 100%;
+  height: 100%;
+  padding: 12px;
+
+  /* text-align: center; */
+}
+
+.contact-form .text-area {
```

Comment:
The `text-area` `font-family` does not match the other `inputs` `font-family`. Please make it consistent across the form.